src/main/resources/views/newCase.rocker.html

```diff
@@ -6,51 +6,51 @@
             <h5>Створити нову справу</h5>
         </div>
         <div class="card-body mt-3">
-            <form>
+            <form action="/cases/new" method="post">
                 <div class="form-group">
                     <label for="currentStage">Стадія розгляду</label>
-                    <input type="text" class="form-control" id="currentStage">
+                    <input type="text" class="form-control" id="currentStage" name="currentStage">
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Дата і час</label>
```

Comment:
In label element, write correspondent column in "for". When specifying id of input, end with "Id" word. For example: "currentStageId".